plugin.json

```diff
@@ -40,9 +40,23 @@
       "name": "Search Bar",
       "description": "Display search bar customized for FINDOLOGIC",
       "defaultLayoutContainer": "Search.SearchBar"
+    },
+    {
+      "key": "Findologic\\Containers\\PromotionContainer",
+      "name": "Promotion",
+      "description": "Display promotion banner",
+      "defaultLayoutContainer": "CategoryItem.Promotion"
     }
   ],
   "dependencies": {
     "findologic/http_request2": "2.3.1"
-  }
+  },
+  "containers": [
+    {
+      "key": "CategoryItem.Promotion",
+      "name": "Category item list: Main container",
```

Comment:
Maybe we should change `Main container` to `Add content to main container`. The current name doesn't give you much info about what the container actually does.